### ISSUE TITLE
support for Swift Package Manager

### ISSUE THREAD

**Issue by appcornerit** [Marxon13/M13Checkbox#142]
Xcode 11 fails with the following error
The package dependency graph can not be resolved; unable find any available tag for the following requirements

**Comment by pariz** [comment]
Feel free to submit a PR on this. I might do it in the near future, it should just be a matter of updating Package.swift to the latest swift tools version.

**Comment by lewis-smith** [comment]
For the time being this won't be possible as SPM only supports code, no assets :( Hopefully this will change in September

**Comment by kumarc-123** [comment]
Xcode 12 has been released and Swift PM supports bundled resources. I think we can add support for Swift PM now.

**Comment by Jeehut** [comment]
Yes, would be great if we could add proper SwiftPM support, I'm getting this error:
```bash
because M13Checkbox >=2.0.0 contains incompatible tools version and root depends on M13Checkbox 3.4.0..<4.0.0, version solving failed.
```

**Comment by that-scalcucci-guy** [comment]
Is this project abandoned? Seems like people have been asking for this for a year now.
